src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', (even) => {
+  const wall = even.target.closest('.wall');
+  const spider = document.querySelector('.spider');
+
+  if (!wall) {
+    return;
+  }
+
+  const wallTop = wall.offsetTop;
+  const wallLeft = wall.offsetLeft;
+
+  const clickX = even.pageX - (wallLeft + spider.offsetWidth / 2);
```

Comment:
Use 'event.clientX' and 'event.clientY' instead of 'event.pageX' and 'event.pageY' as per the task description.